jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/ResourceCollection.java

```diff
@@ -81,10 +81,6 @@ private static void gatherUniqueFlatResourceList(List<Resource> unique, List<Res
                     throw new IllegalArgumentException("Does not exist: " + r);
                 }
 
-                if (!r.isDirectory())
```

Comment:
Why is this line removed in this pr?

Reply:
Basically this restriction was bogus.

The `ResourceCollection` objects used in `MetaInfConfiguration` would be unable to hold the TLD lists then.
This restriction for only directory was a new thing added in Jetty 12, and wasn't present in Jetty 11.